src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureBehaviour {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public abstract String printFigure();
```

Comment:
Create this method also in separate interface `Drawable`, and add  `implements Drawable` in Figure.

Also, let it be just draw()
```suggestion
    public abstract String draw();
```